pkg/config/config.go

```diff
@@ -2,16 +2,16 @@ package config
 
 import (
 	"fmt"
-	"log"
-	"os"
-	"path/filepath"
-	"runtime"
-
 	"github.com/BurntSushi/toml"
 	"github.com/gomods/athens/pkg/download/mode"
 	"github.com/gomods/athens/pkg/errors"
 	"github.com/kelseyhightower/envconfig"
-	validator "gopkg.in/go-playground/validator.v9"
+	"gopkg.in/go-playground/validator.v9"
+	"log"
```

Comment:
can you please run gofmt/goimports so that standard library imports are above and separated from third party dependencies?